modules/serverless-logic-linking-workflows-eventing-system.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-workflow-eventing-system
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-linking-workflows-eventing-system_{context}"]
+= Linking workflows to the eventing system
+
+The {ServerlessLogicOperatorName} links workflows with the eventing system using Knative Eventing, SinkBindings, and triggers. These objects are created automatically by the {ServerlessLogicOperatorName} and simplify the production and consumption of workflow events.
+
+The following example shows the Knative Eventing objects created for an `example-workflow` configured with a platform-scoped eventing system:
+
+.Example of platform-scoped eventing system configuration
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform-example
+  namespace: example-namespace
```

Comment:
User-replaced values should be surrounded by angle brackets.

example-namespace > <example-namespace>